packages/system/gpu-operator/charts/gpu-operator/templates/nodefeaturerules.yaml

```
@@ -0,0 +1,107 @@
+    - name: "TDX total keys rule"
+      extendedResources:
+        tdx.total_keys: "@cpu.security.tdx.total_keys"
+      matchFeatures:
+        - feature: cpu.security
+          matchExpressions:
+            tdx.enabled: {op: IsTrue}
+    - name: "SEV-SNP rule"
+      labels:
+        sev.snp.enabled: "true"
+      matchFeatures:
+      - feature: cpu.security
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)